src/components/TeamMemberList.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react';
+import TeamsMember from '../../public/data/TeamsMember.json';
+import profile from '../../public/profile.svg';
+import MemberProfile from './Modal/MemberProfile';
+
+function TeamMemberList() {
+  const [memberProfile, setMemberProfile] = useState(false);
+  const [memberId, setMemberId] = useState<number | null>(null);
+
+  const handleProfileClick = (userId: number) => {
+    setMemberProfile(!memberProfile);
+    setMemberId(userId);
+  };
+  const { members } = TeamsMember;
+
+  const getMemberId = () => {
+    if (memberId === null) {
+      return null;
+    }
```

Comment:
```suggestion
    if (!memberId) {
      return;
    }
```
이렇게 변경하면 좀 더 깔끔할 것 같아요!